test/Microsoft.DotNet.Darc.VirtualMonoRepo.E2E.Tests/VmrTestsBase.cs

```diff
@@ -32,6 +32,7 @@ internal abstract class VmrTestsBase
     protected NativePath DependencyRepoPath { get; private set; } = null!;
     protected NativePath SyncDisabledRepoPath { get; private set; } = null!;
     protected NativePath InstallerRepoPath { get; private set; } = null!;
+    protected NativePath VmrArcade { get; private set;} = null!;
```

Comment:
Can we name it better? Something like `ArcadeInVmrPath`?
It's a bit weird used later in the code as it sounds like some special arcade VMR.